cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/lib/start-stop-functions

```diff
@@ -0,0 +1,688 @@
+#!/bin/bash
+
+# The original version of this file was taken from the CentOS5
+# initscripts-8.45.30-2.el5.x86_64 package and as this package is licensensed
+# under GPLv2 and GPLv2+ this entire file is provided under those licenses.
```

Comment:
This license requires that we use the same license for the whole solution. Please contact legal for a legal approval.